enaml/scintilla/scintilla.py

```diff
@@ -115,11 +129,11 @@ class Scintilla(Control):
 
     #: The theme to apply to the widget. See the './THEMES' document
     #: for how to create a theme dict for the widget.
-    theme = d_(Typed(dict, ()))
+    theme = d_(Dict())
```

Comment:
The container used by atom for Dict is not  dict subclass which sometimes causes issues. Is this not why it was not used here before ?

Reply:
Not sure why I changed that. Will undo it.